processor/src/main/java/com/loofi/notification/processor/configs/NotificationAppConfig.java

```diff
@@ -2,13 +2,13 @@
 
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
-import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
-import org.springframework.security.crypto.password.PasswordEncoder;
+//import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+//import org.springframework.security.crypto.password.PasswordEncoder;
 
 @Configuration
 public class NotificationAppConfig {
-  @Bean
-  public PasswordEncoder passwordEncoder() {
-    return new BCryptPasswordEncoder();
-  }
+//  @Bean
```

Comment:
- Let's enable this and try to fix
- do few mock ( for sending email + SMS+ PushNotification) ( any mock tools?)
- write down few units or integration test ( testcontainer will help)
- add any other innovative feature for robustness